Firmware/ultralcd.cpp

```diff
@@ -6733,7 +6733,15 @@ static void lcd_rename_sheet_menu()
     {
         lcd_putc(menuData->name[i]);
     }
-    lcd_set_cursor(menuData->selected, 1);
+    if (menuData->selected>0)
+    {
+	    lcd_set_cursor(menuData->selected-1, 1);    
+	    lcd_putc(' ');
```

Comment:
Could be optimised with `lcd_putc_at(menuData->selected-1, 1, ' ');` ;)

Reply:
Does that put the cursor back in the next character position when it is done?

Reply:
Yes exactly, it should do the same. Should save a few bytes to call one function instead of two. Not critical but should save some bytes

Reply:
Done, and further optimized to remove the if/else